app/controllers/posts_controller.rb

```diff
@@ -0,0 +1,76 @@
+# frozen_string_literal: true
+
+class PostsController < ApplicationController
+  before_action :set_post, only: %i[show edit update destroy]
+  before_action :authenticate_user!
+
+  # GET /posts
+  # GET /posts.json
+  def index
```

Comment:
- [ ] Index displays posts in a timeline (recent to older). We can achieve this by using [default scope](https://apidock.com/rails/ActiveRecord/Base/default_scope/class) in `post.rb`. An example could be [this](https://stackoverflow.com/questions/3393687/default-sort-order-for-a-rails-model).